packages/devtools-launchpad/src/index.js

```diff
@@ -93,19 +93,7 @@ function getTargetFromQuery() {
   return null;
 }
 
-function bootstrap(React, ReactDOM, App, appActions, appStore) {
-  const connTarget = getTargetFromQuery();
-  if (connTarget) {
```

Comment:
Yeah good idea. Just did a first pass based on the patch but it'd make more sense to have it do just one thing.